src/App.js

```diff
@@ -1,29 +0,0 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, according to the requirements, you should keep this `App.js` component since it is your "App entry point":

![image](https://user-images.githubusercontent.com/77069011/218862779-e8319856-dee5-4b09-9dc9-1f93fc908b77.png)

Kindly instead of deleting it, import the `<Calculator />` component and use it here! 🤓 🔧